src/shared/types.ts

```diff
@@ -0,0 +1,83 @@
+import {Remote} from 'comlink';
+import {MESSAGE_TYPES} from './constants.js';
+
+declare global {
+  interface Window {
+    __CodeEditorSessions?: Set<string>;
+  }
+  interface ImportMeta {
+    url: string;
+  }
+
+  // interface FetchEvent extends Event {
```

Comment:
why are these commented and not deleted?

Reply:
oops

Reply:
fixed